### PR TITLE
[QQC-362] Document new project update restrictions regarding quality …

### PR DESCRIPTION
**Pull request by richardsun0713**
…mode

### CODE REVIEW

File: labelbox/schema/project.py [hunk 1]
```diff
@@ -93,6 +93,22 @@ class QueueMode(Enum):
         Dataset = "Dataset"
 
     def update(self, **kwargs):
+        """ Updates this project with the specified attributes
+
+        Args:
+            kwargs: a dictionary containing attributes to be upserted
+                * Note that the quality setting cannot be changed after a project has been created. The quality mode
```
Comment (kkim-labelbox) [inline labelbox/schema/project.py:100]: Not sure if github is screwing up the formatting, but could you put the Note on the same indent level as Args?

Comment (jtsodapop) [reply labelbox/schema/project.py:100]: **Question** When is project.update() meant to be used for `auto_audit_number_of_labels` and `auto_audit_percentage` if changing these values is supposed to be an illegal operation? If users cannot change between the two at all, should we jus exclude this information from the docstring?

Comment (richardsun0713) [reply labelbox/schema/project.py:100]: Users can adjust these values when a project is using consensus mode. For example, they might change the number of labels from 2 to 3, and the project's quality setting would remain in consensus mode (thus, a valid operation).

**Review by kkim-labelbox — APPROVED** [review]
LGTM!

**Pull request merged**